tests/unit/test_general_functions.py

```diff
@@ -37,8 +40,8 @@
 
 from modules import general_functions as gf
 from modules.element import Element
-from modules.general_functions import ReferenceDensity
 from modules.layer import Layer
+from modules.reference_density import ReferenceDensity
```

Comment:
`ReferenceDensity` is in a separate file now, so the tests should be in a separate file too (**tests/unit/test_reference_density.py**).

Reply:
OK